src/worker.cpp

```diff
@@ -1217,4 +1220,49 @@ void Worker::collectFrusFromJson()
         }}.detach();
     }
 }
+
+void Worker::performBackupAndRestore(types::VPDMapVariant& io_srcVpdMap)
```

Comment:
This API should be under IBM systems tag? Better club all IBM related APIs together to avoid multiple checking.